TA_Scheduler/views/homepage/home.py

```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+from django.views import View
+
+
+class Home(View):
+    def get(self, request):
+        return render(request, "login/home.html", {})
+
+    def post(self, request):
+        return render(request, "login/home.html", {})
```

Comment:
Currently using `testhome/`

Reply:
I didn't do anything with that . . . Is there something that I need to do with it?